src/store/sagas/authenticatioSaga.js

```diff
@@ -0,0 +1,15 @@
+import { put, call } from 'redux-saga/effects';
+import AuthService from '../../services/AuthService';
+
+import * as types from '../actions'
+
+export function* registerSaga(payload) {
+  try {
+    const response = yield call(AuthService.registerUser, payload);
+    yield [
```

Comment:
Can we use `yield put({ type: types. REGISTER_USER_SUCCESS, error });` here because we don't have multiple tasks that need to be run in parallel?